starboard/shared/posix/socket_connect.cc

```diff
@@ -30,7 +31,7 @@ SbSocketError SbSocketConnect(SbSocket socket, const SbSocketAddress* address) {
 
   sbposix::SockAddr sock_addr;
   if (!sock_addr.FromSbSocketAddress(address)) {
-    SB_LOG(ERROR) << __FUNCTION__ << ": Invalid address";
+    SB_LOG(ERROR) << __FUNCTION__ << ": Invalid address " << *address;
```

Comment:
Is dereferencing `address` always safe here ? I'm assuming so but just double checking - there are no CHECKs that would assert it's not null in the entry here.

Reply:
If it's ever nullptr, it would have already dereferenced it. I'll add a check and wrap the SB_LOG()s in `#if !defined(COBALT_BUILD_TYPE_GOLD)`.